helm_deploy/hmpps-interventions-service/reports/crs_performance_report.sql

```diff
@@ -0,0 +1,47 @@
+COPY (
+  SELECT
+    r.reference_number      AS referral_ref,
+    r.id                    AS referral_id,
+    'coming-later'          AS crs_contract_reference,
```

Comment:
@rgforsyth Just to let you know, I believe this will need to reflect the work in #215 (at some point)